packages/core/nest-application-context.ts

```diff
@@ -131,12 +133,18 @@ export class NestApplicationContext implements INestApplicationContext {
 
   public useLogger(logger: LoggerService | LogLevel[] | false) {
     Logger.overrideLogger(logger);
+
+    if (this.flushLogsOnOverride) this.flushLogs();
```

Comment:
Would you mind updating the code to wrap the internals of the if in curly braces? It leads the code to be easier to follow and modify if we need to

```suggestion
    if (this.flushLogsOnOverride) {
      this.flushLogs();
    }
```

Reply:
I prefer your way tbh. I thought `npm run lint:fix` would fix that :p